### DISCUSSION TITLE
passwordless auth

### DISCUSSION THREAD

**Discussion by wesley974** [pocketbase/pocketbase#2660] [Q&A]
Hello,

I need backend with passwordless auth. Does it provide this feature?

Thank's!

**Comment by ganigeorgiev** [comment]
No, there is no such auth method at the moment.

The closest supported option at the moment without explicitly requiring users to enter a password is the OAuth2 authentication (_you can find all available providers in the "Admin UI > Settings > Auth providers"; in the [`develop` branch](https://github.com/pocketbase/pocketbase/tree/develop) there are also Instagram and VKontakte clients that will be available with the next release_).
